lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -722,7 +722,7 @@ void TypeAnalysis::visitOperation(Operation *op,
 
   // Promote the two dtypes assuming possibly-zero rank.
   if (isa<AtenAddTensorOp, AtenSubTensorOp, AtenMulTensorOp, AtenDivTensorOp,
-          AtenDivTensorModeOp, Aten__And__TensorOp, AtenMinimumOp,
+          AtenDivTensorModeOp, Aten__And__TensorOp, AtenMinimumOp, AtenAtan2Op,
```

Comment:
Why are there two cases for `AtenAtan2Op` in this file? This makes it a bit harder to know how a single op is being handled

Reply:
Oh I forgot to remove the first one.